docs/the-guide/chapter-1-run-a-simple-ml-experiment/index.md

```diff
@@ -169,6 +169,16 @@ poetry install
 poetry shell
 ```
 
+!!! question
+
+	**Why Poetry?**
+
+	Poetry is a tool to manage Python dependencies. It is a more robust and
+	user-friendly alternative to `pip`. It is also more suitable for
+	reproducibility and collaboration by creating a lock file that can be used
+	to recreate the exact same environment.
+
+
```

Comment:
You can remove the double empty lines here (with the future formatter/linter, it shouldn't be an issue anymore :)).